core/src/epicli/data/common/ansible/playbooks/roles/postgresql/tasks/main.yml

```diff
@@ -1,141 +1,150 @@
       Debian:
-        - postgresql-10
-        - postgresql-contrib-10
-        - python-psycopg2 # required for postgresql ansible management
+        - postgresql-client-common
+        - postgresql-common
+        - postgresql-client-13
+        - postgresql-13
+        - postgresql-contrib-13
+        - python-psycopg2 # required for PostgreSQL ansible management
       RedHat:
-        - postgresql10-server
-        - python-psycopg2 # required for postgresql ansible management
+        - postgresql13-server
+        - python-psycopg2 # required for PostgreSQL ansible management
   module_defaults:
     yum: { lock_timeout: "{{ yum_lock_timeout }}" }
 
 # Extension packages need to be installed first to be able to process PostgreSQL config
 - name: Install extension packages
-  include_tasks: extensions/{{ data.name }}/packages.yml
+  include_tasks: extensions/{{ extension }}/packages.yml
   loop_control:
-    loop_var: data
+    loop_var: extension
   loop: "{{ enabled_extensions }}"
 
-- name: Create directory /var/log/postgresql
-  # The same permissions as for PG installed on RHEL from Software Collections (different on Ubuntu)
+- name: Create directories and ensure correct permissions
   file:
-    path: /var/log/postgresql
+    path: "{{ path }}"
     state: directory
     owner: postgres
     group: postgres
     mode: u=rwx,g=,o=
+  loop_control:
+    loop_var: path
+  loop:
+    - /var/log/postgresql
+    - "{{ pg.data_dir[ansible_os_family] }}" # Permissions should be u=rwx (0700) or u=rwx,g=rx (0750)
 
 - name: RedHat | Initialize database
+  when: ansible_os_family == 'RedHat'
   block:
-    - name: Check initialization status (RedHat)
+    - name: RedHat | Check initialization status
       stat:
         path: "{{ pg.config_dir[ansible_os_family] }}/pg_hba.conf"
       register: pg_hba_conf_stat
 
-    - name: Initialize database (RedHat)
-      command: /usr/pgsql-10/bin/postgresql-10-setup initdb {{ pg.service_name[ansible_os_family] }}
-      when:
-        - not pg_hba_conf_stat.stat.exists
-  when:
-    - ansible_os_family == 'RedHat'
-
-- name: Change pg_hba.conf
-  replace:
-    path: "{{ pg.config_dir[ansible_os_family] }}/pg_hba.conf"
-    regexp: '^host\s+all\s+all\s+127\.0\.0\.1/32\s+(?:ident|md5)$'
-    replace: 'host    all             all             0.0.0.0/0               md5'
-    backup: true
-  register: change_pg_hba_conf
+    - name: RedHat | Initialize database
+      become_user: postgres
+      when: not pg_hba_conf_stat.stat.exists
+      command: "{{ pg.bindir[ansible_os_family] }}/initdb"
+      environment:
+        PGDATA: "{{ pg.data_dir[ansible_os_family] }}"
+
+- name: Ensure that PostgreSQL service is started
+  block:
+    - name: Ensure that PostgreSQL service is started
+      systemd:
+        name: "{{ pg.service_name[ansible_os_family] }}"
+        state: started
+        enabled: true
+
+    - &check-debian-service
+      name: Debian | Check that the main service is started
```

Comment:
In 13th version it's also so, but this service doesn't care about its children state/status.

```text
cat /etc/systemd/system/multi-user.target.wants/postgresql.service
# systemd service for managing all PostgreSQL clusters on the system. This
# service is actually a systemd target, but we are using a service since
# targets cannot be reloaded.
[Unit]
Description=PostgreSQL RDBMS
[Service]
Type=oneshot
ExecStart=/bin/true
ExecReload=/bin/true
RemainAfterExit=on
[Install]
WantedBy=multi-user.target
```